lib/DDG/Spice/Syntax.pm

```diff
@@ -0,0 +1,31 @@
+package DDG::Spice::Syntax;
+
+# ABSTRACT: Look up syntax for a programming language
+
+use DDG::Spice;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
+spice to => 'https://syntaxdb.com/api/v1/concepts/search?q=$1';
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers any => 'syntax', 'syntaxdb', 'syntaxcenter';
+my $languages = join "|", share('languages.txt')->slurp;
```

Comment:
You'll want to do `join '|', map { quotemeta $_ } ...` so we don't mess up the regexes.